packages/govuk-frontend-review/package.json

```diff
@@ -45,6 +45,7 @@
     "gulp-cli": "^2.3.0",
     "nodemon": "^2.0.22",
     "postcss-pseudo-classes": "^0.2.1",
+    "puppeteer": "^20.2.0",
```

Comment:
We forgot to move **percy.config.js** in https://github.com/alphagov/govuk-frontend/pull/3620 which needs puppeteer as a `devDependency`